vmdb/app/models/configured_system_foreman.rb

```diff
@@ -1,2 +1,13 @@
 class ConfiguredSystemForeman < ConfiguredSystem
+  include ProviderObjectMixin
+
+  def provider_object(connection = nil)
+    (connection || raw_connect).host_ref(manager_ref)
```

Comment:
I find the method name host_ref confusing as it returns an object...whereas the pattern for _ref (like manager_ref, ems_ref), is to return the provider's unique id for that object.  It's especially evident here where both words are used.